Gitattributes update for silencing "modified" pdfs

abottoms-coder opened the pull request:
### Summary of Changes
_Provide a brief summary of changes_
_Addresses issue #1235
_Acceptance criteria as stated in the issue_
Prevent pdfs showing as modified.

### How to Test
_List the steps to test the PR_
On a clean checkout of raft-tdp-main, run `git status`
You may see the modified files as per the ticket's screenshot.

Switch to this branch hotfix/1235-pdfs-showing-modified and again run `git status` to see that pdfs do not show as modified.

ADPennington commented:
@abottoms-coder i've not been able to detect the pdf issue. see terminal commands below:

```
alexandrapennington@Alexandras-MacBook-Pro TANF-app % git checkout raft-tdp-main
Switched to branch 'raft-tdp-main'
Your branch is up to date with 'origin/raft-tdp-main'.
alexandrapennington@Alexandras-MacBook-Pro TANF-app % git status
On branch raft-tdp-main
Your branch is up to date with 'origin/raft-tdp-main'.

nothing to commit, working tree clean
alexandrapennington@Alexandras-MacBook-Pro TANF-app % 
```

abottoms-coder commented:
@ADPennington 
While it may not occur in your Mac environment, this is affecting 2 or 3 of our dev team. I'd like to get this merged even if the problem is not reproducible on your end as it is for others. 

```
aj@raft-dev:~/Documents$ mkdir pdfbroke
aj@raft-dev:~/Documents$ cd pdf
bash: cd: pdf: No such file or directory
aj@raft-dev:~/Documents$ cd pdfbroke/
aj@raft-dev:~/Documents/pdfbroke$ git clone git@github.com:raft-tech/TANF-app.git
Cloning into 'TANF-app'...
remote: Enumerating objects: 20998, done.
remote: Counting objects: 100% (3181/3181), done.
remote: Compressing objects: 100% (1398/1398), done.
remote: Total 20998 (delta 2110), reused 2635 (delta 1691), pack-reused 17817
Receiving objects: 100% (20998/20998), 21.48 MiB | 22.58 MiB/s, done.
Resolving deltas: 100% (14207/14207), done.
aj@raft-dev:~/Documents/pdfbroke$ cd TANF-app/
aj@raft-dev:~/Documents/pdfbroke/TANF-app$ git status
On branch raft-tdp-main
Your branch is up to date with 'origin/raft-tdp-main'.

Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git restore <file>..." to discard changes in working directory)
	modified:   docs/Acquisition/COR Delegation Robert Jolly.pdf
	modified:   docs/Acquisition/COR_Delegation_Dvora_Wilensky.pdf
	modified:   docs/User-Research/2020, Fall - Concept Prototypes/TANF Uploading Tool_Mockup #1.pdf
	modified:   docs/User-Research/2020, Fall - Concept Prototypes/TANF Uploading Tool_Mockup #2.pdf

no changes added to commit (use "git add" and/or "git commit -a")
aj@raft-dev:~/Documents/pdfbroke/TANF-app$ date
Thu 19 Aug 2021 10:26:04 AM EDT
aj@raft-dev:~/Documents/pdfbroke/TANF-app$
```

ADPennington reviewed (approved):
skipping qasp checklist for this one. impacting DX but no impact on TDP. 👍🏾 @abottoms-coder ready to merge.
